config/routes.rb

```diff
@@ -7,7 +7,11 @@
   authenticate :admin_user do
     ExceptionHunter.routes(self)
   end
-  ActiveAdmin.routes(self)
+  begin
```

Comment:
```
ActiveAdmin::DatabaseHitDuringLoad: Your file, app/admin/file_ignoring_rules.rb (line 15), caused a database error while Active Admin was loading. This is most common when your database is missing or doesn't have the latest migrations applied. To prevent this error, move the code to a place where it will only be run when a page is rendered. One solution can be, to wrap the query in a Proc. Original error message: FATAL:  database "github_analyzer_development" does not exist
```

I think the correct solution is using a proc instead of adding this catch logic

Reply:
Fixed @santib! could you take a look?